plugins/scaffolder-annotator-action/package.json

```diff
@@ -54,14 +49,14 @@
   },
   "devDependencies": {
     "@backstage/backend-common": "0.23.3",
```

Comment:
```suggestion
```

Reply:
I recommend double-checking to ensure that all the dependencies are truly necessary.

Reply:
Pardon! Thank you, updated.